webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/jdbc/EventLogJdbcRepository.scala

```diff
@@ -287,6 +252,48 @@ class EventLogJdbcRepository(
   }
 }
 
+object EventLogJdbcRepository {
+
+  object sql {
+
+    // Query to get Last event by change request, we need to do something like a groupby and get the one with a higher creationDate
+    // We partition our result by id, and order them by creation desc, and get the number of that row
+    // then we only keep the first row (rownumber <=1)
+    def eventLogQuery(
```

Comment:
I'm not sure that nomenclature make things clearer. 
You're already in a "Jdbc" thing, it's awaited that you will deal with sql. On the other hand, there is a lot of event log related queries in that repos, so here we don't really know which one you are talking about. 
Can you remove the inner object and just call the method `getLastEventByChangeRequestSQL` or something?